src/openrct2/object/RideObject.cpp

```diff
@@ -124,7 +124,7 @@ void RideObject::ReadLegacy(IReadObjectContext* context, IStream* stream)
     }
 
     // Read peep loading positions
-    for (int32_t i = 0; i < RCT2_MAX_VEHICLES_PER_RIDE_ENTRY; i++)
+    for (int32_t i = 0; i < RCT2::Limits::MaxVehiclesPerRideEntry; i++)
```

Comment:
I wonder if we should call these "RCT2 limits" or "DAT limits". I think the latter might be clearer.

Reply:
Yeah I'm of a similar mind. It will be easy enough to move things about after this pr.